java/vote/pom.xml

```diff
@@ -26,9 +26,9 @@
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
 		 <dependency>
-            <groupId>io.diagrid.dapr</groupId>
-            <artifactId>dapr-spring-boot-starter</artifactId>
-            <version>0.10.7</version>
+            <groupId>io.dapr</groupId>
```

Comment:
@ThomasVitale any reason why you changed this? If you just use the SDK you cannot run the local development mode? Is there something in the SDK (like the version) that you were not getting using the dapr-spring-boot-starter?

Reply:
The starter is bringing the Dapr Java SDK, but also the Dapr Spring Boot SDK which is based on Spring Boot 2, bringing dependencies that don't work anymore but also resulting in the auto configuration not to work (indeed, you cannot autowire a DaprClient, but you need to build one yourself). Also, it's bringing several test dependencies (including Testcontainers) in the compile scope that end up in the final production artifact.

I haven't included that part in this PR, but for the dev loop we would add the starter in the test scope to get the Testcontainers support. What do you think?